docs/howto/manage-storage.md

```diff
@@ -2,136 +2,68 @@
 # How to manage storage
 > See first: {external+juju:ref}`Juju | Storage <storage>`, {external+juju:ref}`Juju | Manage storage <manage-storage>`, {external+charmcraft:ref}`Charmcraft | Manage storage <manage-storage>`
 
-## Implement the feature
+## Manage storage for a machine charm
 
-### Declare the storage
+### Define the storage
 
-To define the storage that can be provided to the charm, define a `storage` section in `charmcraft.yaml` that lists the storage volumes and information about each storage. For example, for a transient filesystem storage mounted to `/cache/` that is at least 1GB in size:
+Define a `storage` section in `charmcraft.yaml` that lists the storage volumes and information about each storage. For example, for a transient filesystem storage mounted to `/cache/` that is at least 1GB in size:
```

Comment:
I wonder if we should leave out the location entirely. It's not particularly useful to the charm, because the charm should always get the location from Juju, and it leads to bad behaviour where the charm hard-codes paths based on this instead of asking Juju.